src/H5FDsubfiling/H5FDsubfiling.c

```diff
@@ -0,0 +1,3264 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
```

Comment:
This is the Subfiling VFD, which has the main purpose of breaking I/O requests down into vector I/O requests according to how the I/O requests fall across the different subfiles that the logical HDF5 file is being striped across. It then sends the vector I/O requests down to the underlying I/O concentrator, which handles the details of writing the I/O portions to each subfile.